src/pages/events-description/index.js

```diff
@@ -0,0 +1,198 @@
+import EventsStyleDescription from './styled'
+import { useState, useEffect } from 'react'
+import { cms } from '../../service/client'
+import { useParams } from 'react-router'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import { FreeMode, Navigation, Thumbs } from 'swiper/modules'
+import { Link } from 'react-router-dom'
+import env from 'react-dotenv'
+import 'swiper/css'
+import 'swiper/css/free-mode'
+import 'swiper/css/navigation'
+import 'swiper/css/thumbs'
+import {
+  faRectangleXmark,
+  faUser,
+  faPenToSquare,
+  faLock,
+  faHandHoldingDollar,
+  faCalendarDays,
+  faLocationDot
+} from '@fortawesome/free-solid-svg-icons'
+
+export const EventsPageDescription = () => {
+  const [event, setEvent] = useState([])
+  const [thumbsSwiper, setThumbsSwiper] = useState(null)
+  const [galeria, setGaleria] = useState([])
+  const { id } = useParams()
+  useEffect(() => {
+    cms.get(`api/events/${id}/?populate=parceires&populate=fotos_evento`).then((response) => {
+      const { data } = response.data
+      setEvent(data)
+
+      const fotoDivulgacao = data.attributes.fotos_evento
+      const images = fotoDivulgacao.data.map(
+        (image) => {
+          return {
+            id: image.id,
+            name: image.attributes?.name,
+            url: env.URL_CMS + image.attributes?.url
+          }
+        }
+      )
+      setGaleria(images)
+    })
+  }, [])
+  const handleDate = (date) => {
+    const day = date.toLocaleDateString(undefined, { day: 'numeric', Timezone: 'UTF' })
+    const month = date.toLocaleDateString('pt-BR', { month: 'long' })
+    const year = date.toLocaleDateString(undefined, { year: 'numeric', Timezone: 'UTF' })
+    return [day, month, year].join(' ')
+  }
+  const dataEmObjDate = new Date(event?.attributes?.data)
+  const indexWeek = dataEmObjDate.getDay()
```

Comment:
Sugeriria manter a nomenclatura de variáveis sempre em inglês
```suggestion
  const dateAsDateObject = new Date(event?.attributes?.data)
  const indexWeek = dateAsDateObject.getDay()
```